audit-service/src/test/java/uk/gov/gchq/palisade/service/audit/service/AuditServiceTestCommon.java

```diff
@@ -114,7 +114,7 @@ static Rules mockRules() {
                 .withResourceId(mockResource().getId())
                 .withContext(mockContext())
                 .withException(mockException())
-                .withServiceClass(Service.class);
+                .withServiceName("Service");
```

Comment:
We shouldn't ever have a registerRequestException with the service name being "Service". It should be split into 4 methods with the service coming from Palisade service, User service, Resource service and Policy service. These should also use the Enum class

Reply:
`ServiceName` is in `palisade-service`. Should the enum be copied or the enum moved to common?

Reply:
I'm not sure where the 4 methods come in, but that method has been changed.

Reply:
done